modules/virt-example-nmstate-IP-management.adoc

```diff
@@ -91,33 +91,156 @@ The following snippet configures an Ethernet interface that uses a dynamic IP ad
-        search:
-        - example.com
-        - example.org
-        server:
-        - 8.8.8.8
+dns-resolver:
+  config:
+    search:
+      - example.com
+      - example.org
 # ...
 ----
-<1> You must configure an interface with `auto-dns: false` or you must use static IP configuration on an interface in order for Kubernetes NMState to store custom DNS settings.
+
+[TIP]
+====
+Setting a DNS configuration is comparable to modifying the `/etc/resolv.conf` file.
+====
+
+The following examples show situations that require configuring a network interface to store DNS values:
 
 [IMPORTANT]
 ====
-You cannot use `br-ex`, an OVNKubernetes-managed Open vSwitch bridge, as the interface when configuring DNS resolvers.
+You cannot use `br-ex` bridge, an OVNKubernetes-managed Open vSwitch bridge, as the interface when configuring DNS resolvers.
 ====
 
+* Configure a static DNS for a network interface with an automatic IP configuration. Note that for this configuration, you must set the `auto-dns` parameter to `false`, so that the Kubernetes NMState Operator can store custom DNS settings for the network interface.
++
+[source,yaml]
+----
+# ...
+    server:
+    - 2001:db8:f::1
+    - 192.0.2.251
+interfaces:
+  - name: eth1
+    type: ethernet
+    state: up
+    ipv4:
+      enabled: true
+      dhcp: true
+      auto-dns: false
+    ipv6:
+      enabled: true
+      dhcp: true
+      autoconf: true
+      auto-dns: false
+# ...
+----
+
+* Configure a static DNS for a network interface with an static IP configuration. Note that for this configuration, you must set the `auto-dns` parameter to `true`.
++
+[source,yaml]
+----
+# ...
+    server:
+    - 2001:4860:4860::8844
+    - 192.0.2.251
+interfaces:
+  - name: eth1
+    type: ethernet
+    state: up
+    ipv4:
+      enabled: true
+      dhcp: false
+      address:
+      - ip: 192.0.2.251
+        prefix-length: 24
+    ipv6:
+      enabled: true
+      dhcp: false
+      autoconf: false
+      address:
+      - ip: 2001:db8:1::1
+        prefix-length: 64
+routes:
+  config:
+  - destination: 0.0.0.0/0
+    next-hop-address: 192.0.2.1
+    next-hop-interface: eth1
+  - destination: ::/0
+    next-hop-address: 2001:db8:1::3
+    next-hop-interface: eth1
+# ...
+----
+
+* Configure a static DNS name server to append to a The Dynamic Host Configuration Protocol (DHCP) server:
++
+[source,yaml]
+----
+# ...
+    server:
+    - 127.0.0.1
+interfaces:
+  - name: eth1
+    type: ethernet
+    state: up
+    ipv4:
+      enabled: true
+      dhcp: true
+      auto-dns: true
+    ipv6:
+      enabled: true
+      dhcp: true
+      autoconf: true
+      auto-dns: true
+# ...
+----
++
+[NOTE]
+====
+To configure a static DNS name server to search for a suitable DNS server, do note define the `server` and `auto-dns` parameter in the configuration file.
+====
+
+* Configure a static IPv6 DNS server that links to a local server:
++
+[source,yaml]
+----
+# ...
+    server:
+    - fe80::deef:1%eth1
```

Comment:
This configuration is similar to the "Configure a static DNS for a network interface with an static IP configuration" situation with the exeption of an extra defined server, so can I delete this example and just a note under the "Configure a static DNS for a network interface with an static IP configuration" example?